index.Rmd

```diff
@@ -3,19 +3,26 @@ title: "R Development Guide"
 author: "R Contribution Working Group"
 date: "`r Sys.Date()`"
 site: bookdown::bookdown_site
+output: bookdown::gitbook
 documentclass: book
 bibliography: [book.bib]
 biblio-style: apalike
 link-citations: yes
 github-repo: forwards/rdevguide
 url: 'TDDO'
+cover-image: "img/Rlogo.png"
+favicon: "img/Rlogo.svg"
 description: "A guide to R development."
 ---
 
-# {-}
 
-Initial chapters of this guide were developed by Saranjeet Kaur Bhogal, in a project funded by the R Foundation, mentored by Heather Turner and Michael Lawrence. It draws on documentation and articles written by the R Core Team. 
+# Acknowledgement
 
-This guide has benefited from contributions and reviews by the following (names are in alphabetical order):
+Initial chapters of this guide were developed by Saranjeet Kaur Bhogal, in a project funded by the R Foundation, mentored by Heather Turner and Michael Lawrence. It draws on documentation and articles written by the R Core Team. The initial version of the guide was upgraded as a part of the [Google Season of Docs 2022](https://github.com/rstats-gsod/gsod2022/wiki/GSOD-2022-Proposal) project with two Technical Writers and reviewed by the [Steering Committee](https://github.com/rstats-gsod/gsod2022/wiki/GSOD-2022-Proposal#steering-committee).
 
-Bettina Gruen, Carol Willing, Gabriel Becker, Kara Woo, Lluís Revilla Sancho, Maya Gans, Michael Chirico, Mine Cetinkaya-Rundel, Toby Dylan Hocking.
+This guide has benefited and continues to be benefited from varied contributions by several [contributors](https://github.com/r-devel/rdevguide#contributors-). 
+
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/){width=180px}
```

Comment:
Use the images from Creative Commons: https://creativecommons.org/choose/results-one?license_code=by&amp;jurisdiction=&amp;version=4.0&amp;lang=en. The image must be included in this repo (in the `./img` directory) as a PNG or PDF so that the book will compile to PDF.